### DISCUSSION TITLE
Getting Internal Server Error on List Deployment Files endpoint

### DISCUSSION THREAD

**Discussion by mr-loop-1** [vercel/community#3163] [Help]
### Summary

I created a token and successfully fetched my **deployment id** through my deployment url
```
/v13/deployments/{url}
```
But when I am hitting the [List Deployment Files](https://vercel.com/docs/rest-api/endpoints#list-deployment-files) endpoint, it is giving me 500: Internal Server Error as response after a long wait.
```
/v6/deployments/{id}/files
```

I even check using the cUrl version but its the same
`{"error":{"code":"internal_server_error","message":"An unexpected internal error occurred"}}`

Can anybody point to some solution

### Steps to Reproduce

```
curl -X GET "https://api.vercel.com/v6/deployments/<deployment id>/files" -H "Authorization: Bearer <Token>"
```

**Comment by amyegan** [comment]
Hi @mr-loop-1. I don't get a 500 error for that endpoint now. Are you still running into this error?

**Reply by mr-loop-1** [reply]
Yes, I checked it just now. The id was being corectly fetched but 500 on files

**Comment by umarz317** [comment]
Hey, i'm also getting this issue can you please guide me on this? @mr-loop-1 @amyegan
